docs/getting-started/licensing/license-key-and-activation.md

```diff
@@ -99,14 +99,14 @@ A token URL and other authentication methods will be assigned to this specific e
 
 ### Trial license key
 
-This key grants access to **all features**. It is valid for **14 days**. It does not consume editor loads, but editor is limited functionally (for example: session time, number of changes). It is **perfect for evaluating the platform** and all its features. It can be used only for evaluation purposes.
+This key grants access to **all features**. It is valid for **14 days**. It does not consume editor loads if you are on the usage-based billing plan, but editor is limited functionally (for example: session time, number of changes). It is **perfect for evaluating the platform** and all its features. It can be used only for evaluation purposes.
```

Comment:
```suggestion
This key grants access to **all features.** It is valid for **14 days.** It does not consume editor loads if you are not on the usage-based billing plan. However, it limits the editor's functionality (like session time or number of changes). It is perfect for evaluating the platform and all its features. The key exists only for evaluation purposes.
```

Reply:
It is a good practice to put punctuation marks in bold when the adjacent word is in bold.